docs/application/native/guides/connectivity/connection.md

```diff
@@ -154,6 +154,55 @@ To get the type of the current connection, IP address, and proxy information:
     }
     ```
 
+<a name="info"></a>
```

Comment:
The same name is already used on line 206. 
Please assign different name, or use "getting-internet-connection-state-information" which is automatically created for this section. (In that case, you don't need to set name.)

Also how about editing the line 37, too?

**Current :**   You can \[gather various statistics on the network usage](#info), such as the amounts of sent and received data, ...

**For example to :** You can gather various statistics on the network usage, such as the \[connection state](#getting-internet-connection-state-information), and \[amounts of sent and received data](#info), ...